coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -5485,11 +5485,8 @@ def rand_like(context, node):
 
 @register_torch_op
 def randn(context, node):
-    inputs = _get_inputs(context, node, expected=[5, 6])
-
+    inputs = _get_inputs(context, node, expected=1)
```

Comment:
Thanks for creating this PR! Could you run those randn-related tests locally? I believe this will break existing tests.